src/externalized/ExplosiveModel.h

```diff
@@ -23,41 +51,49 @@ struct ExplosiveModel : ItemModel
 		int8_t reliability,
 		int8_t repairEase,
 		uint32_t flags,
-		uint8_t damage,
-		uint8_t stunDamage,
-		uint8_t radius,
 		uint8_t noise,
 		uint8_t volatility,
-		uint8_t type,
+		bool isPressureTriggered,
+		const ExplosiveBlastEffect *blastEffect,
+		const ExplosiveStunEffect *stunEffect,
+		const ExplosiveSmokeEffect *smokeEffect,
+		const ExplosiveLightEffect *lightEffect,
```

Comment:
What do you think about using std::optional for these four member variables? Just an idea, not terribly important. It would save a bit of manual memory management.